packages/ui-extensions-server-kit/.npmignore

```diff
@@ -2,3 +2,4 @@
 node_modules
 src
 tsconfig.*
+*.config.ts
```

Comment:
We were including unnecessary files on the npm tarball (e.g. jest.config.ts and vite.config.ts). This removes them.